src/LCT.PackageIdentifier/DebianProcessor.cs

```diff
@@ -104,8 +104,10 @@ public static Bom RemoveExcludedComponents(CommonAppSettings appSettings, Bom cy
             foreach (var component in componentsForBOM)
             {
                 string repoName = GetArtifactoryRepoName(aqlResultList, component, bomhelper);
-                string jfrogpackageName = $"{component.Name}-{component.Version}{ApiConstant.DebianExtension}";
-                var hashes = aqlResultList.FirstOrDefault(x => x.Name == jfrogpackageName);
+                string jfrogpackageName = $"{component.Name}_{component.Version}";
+                var hashes = aqlResultList.FirstOrDefault(x => x.Name.Contains(
```

Comment:
You can add some logs, so that we don't have to check every time. what's the found Repo details for the particular package name

Reply:
done